Tool test topology build failed

m4: test-tone-playback.m4: 45: fatal error: Invalid parameters (8) to PIPELINE_ADD
m4: test-tone-playback.m4: 45: fatal error: Invalid parameters (8) to PIPELINE_ADD
m4: test-tone-playback.m4: 45: fatal error: Invalid parameters (8) to PIPELINE_ADD
test/topology/CMakeFiles/tests.dir/build.make:106: recipe for target 'test/topology/CMakeFiles/tests' failed
make[3]: *** [test/topology/CMakeFiles/tests] Error 123
CMakeFiles/Makefile2:1557: recipe for target 'test/topology/CMakeFiles/tests.dir/all' failed
make[2]: *** [test/topology/CMakeFiles/tests.dir/all] Error 2
CMakeFiles/Makefile2:1564: recipe for target 'test/topology/CMakeFiles/tests.dir/rule' failed
make[1]: *** [test/topology/CMakeFiles/tests.dir/rule] Error 2
Makefile:697: recipe for target 'tests' failed
make: *** [tests] Error 2

--- a/tools/test/topology/test-tone-playback.m4
+++ b/tools/test/topology/test-tone-playback.m4
@@ -44,7 +44,7 @@ define(`TONE_SAMPLE_RATE', 48000)
 # Schedule with 48 frame per 1000us deadline on core 0 with priority 0
 PIPELINE_ADD(sof/pipe-tone.m4,
 	5, 2, TEST_PIPE_FORMAT,
-	48, 1000, 0, 0)
+	48, 1000, 0, 0, 0)
 
 # playback DAI is SSP2 using 2 periods
 # Buffers use TEST_DAI_FORMAT format, with 48 frame per 1000us on core 0 with priority 0

